Test/Platform/Tests/CLR/mscorlib/systemlib/systemlib2/SystemDateTimeTests.cs

```diff
@@ -1941,11 +1944,11 @@ public MFTestResults DateTime_TicksTest52()
             try
             {
                 Log.Comment("Creating a DateTime, getting the Ticks and Verifying");
-                DateTime testDateTime = new System.DateTime(0);
+                DateTime testDateTime = new System.DateTime(635688622030533832);

```

Comment:
OK, so a named const with a comment explaining that would help clarify that point to so no-one else needs to wonder...